src/System.Net.HttpListener/src/System/Net/Windows/CookieExtensions.cs

```diff
@@ -17,11 +17,7 @@ public static string ToServerString(this Cookie cookie)
             if (s_toServerStringFunc == null)
             {
                 BindingFlags flags = BindingFlags.Instance;
-#if uap
-                flags |= BindingFlags.Public;
-#else
                 flags |= BindingFlags.NonPublic;
```

Comment:
As in the other cookies case, this can be:
```C#
const BindingFlags Flags = BindingFlags.Instance | BindingFlags.NonPublic;
```
or just use the `BindingFlags.Instance | BindingFlags.NonPublic` at the call site.

Same below.